scripts/runtime/VibeRuntime.Common.ps1

```
@@ -3,6 +3,45 @@ $ErrorActionPreference = 'Stop'
 
 . (Join-Path $PSScriptRoot '..\common\vibe-governance-helpers.ps1')
 
+$retiredConsultationHelper = Join-Path $PSScriptRoot 'legacy\VibeRetiredConsultation.Common.ps1'
+$script:VibeRetiredConsultationHelperMissingMessage = "Missing retired consultation helper: $retiredConsultationHelper"
+if (Test-Path -LiteralPath $retiredConsultationHelper -PathType Leaf) {
+    . $retiredConsultationHelper
+}
+if (-not (Get-Command -Name New-VibeRetiredSpecialistConsultationLifecycleLayerProjection -CommandType Function -ErrorAction SilentlyContinue)) {
+    function New-VibeRetiredSpecialistConsultationLifecycleLayerProjection {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'New-VibeRetiredSpecialistConsultationLifecycleLayerProjection' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)